### ISSUE TITLE
iis node issue on APIM

### ISSUE THREAD

**Issue by kieran-clifton-civica** [Azure/api-management-developer-portal#2273]
**Every bug report should have precise description and reproduction steps; console traces or source code references are appreciated.**

**For assistance requests, contact Azure support or submit a post [on Stack Overflow](http://aka.ms/apimso). We don't provide support through GitHub Issues. Feature requests can be raised [on the Azure Feedback Forum](https://aka.ms/apimwish).**

## Bug description

Getting a iisnode error when I open the Azure API managed developer portal 

iisnode encountered an error when processing the request.

![image](https://github.com/Azure/api-management-developer-portal/assets/115151875/c4c72622-4e04-47d8-8995-67c904ce7569)

## Expected behavior

This is a new issue, when I was using this yesterday this issue wasnt here

## Is your portal managed or self-hosted?

Managed 

## API Management service name

https://integration-fsa-rof-gateway.developer.azure-api.net/

## Environment

 - Operating system: [Windows]
 - Browser: [Chrome & Edge]
 - Version: [latest]

**Comment by maciejtreder** [comment]
@kieran-clifton-civica  Thank you for reporting that. I found the logs that are related to this issue. Looks like this is related to the one of the new features: custom access. Have you make any changes in the restricting access to some pages/elements on the page? If yes, what kind?

**Comment by kieran-clifton-civica** [comment]
@maciejtreder thanks for your reply and thanks for looking into this for me, I did change a feature like that yes and then reverted it. 
I would change it but I cant access the Azure developer Portal at all to edit anything because that error page is the only page available now - is there a way to fix this without accessing the APIM portal?

**Comment by maciejtreder** [comment]
@kieran-clifton-civica Just applied the fix on your database. Please try to republish the portal now, using the buttom available in the portal.azure.com

**Comment by kieran-clifton-civica** [comment]
@maciejtreder I have re-published, issue still present

**Comment by maciejtreder** [comment]
@kieran-clifton-civica Yeah. You're right, my fault. I forgot that for publishment you need to have properly running back-end. I've just manually upload a fix to your instance. It is working now. Please remember that this is temporary fix, and it might be gone if it will not be included in the upcoming release.

**Comment by kieran-clifton-civica** [comment]
@maciejtreder brilliant, thank you so much! I will try and not break it in the mean time

**Comment by mrcarlosdev** [comment]
This issue is related to managed developer portal. We advise you to create a Azure support request to get assistance on this issue. Please refer to the below link to create a new Azure support request, Please select Problem Type = "Developer Portal" in the request to route it appropriately.

https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request
